editoast/src/client/user.rs

```diff
@@ -87,3 +97,35 @@ pub async fn add_user(args: AddArgs, pool: Arc<DbConnectionPoolV2>) -> anyhow::R
     println!("User added with id: {}", subject_id);
     Ok(())
 }
+
+/// Get a user
+pub async fn get_user(
+    GetArgs { identity }: GetArgs,
+    openfga_config: OpenfgaConfig,
+    pool: Arc<DbConnectionPoolV2>,
+) -> anyhow::Result<()> {
+    let regulator = openfga_config.into_regulator(pool).await?;
+    let driver = regulator.driver();
+    let Some(User {
+        id,
+        info: UserInfo { identity, name },
+    }) = driver.get_user_info_by_identity(&identity).await?
+    else {
+        println!("User not found");
```

Comment:
```suggestion
        tracing::error!(?identity, "User not found");
```

Reply:
Done in https://github.com/OpenRailAssociation/osrd/compare/b5bcb14002444f3ba5aa2a45718fa551031fde65..889d5631774d96bf9a508f3946d11693cd89e8a7.